R/translate.R

```diff
@@ -80,12 +89,11 @@ deepl_translate <- function(path,
     )
 
     markdown_lines[shortcodes_no] <- sprintf("`%s`", purrr::map_chr(shortcodes, digest::digest))
-    brio::write_lines(markdown_lines, temp_markdown_file)
```

Comment:
I still don't understand why these lines have to go. Especially the writing to the temp file.

Reply:
(see previous)